spec/lib/audit/moab_to_catalog_spec.rb

```diff
@@ -196,7 +196,7 @@
       it "call #create if object does not exist" do
```

Comment:
super nitpicky, but would you mind updating this to `#create_with_validation`?

Reply:
I would be happy to - missed that, and also strongly prefer it to match the actual test code.